frontend/src/static/js/main.js

```diff
@@ -23,6 +23,7 @@ window.app = new window.Vue({
     loadedRepo: 0,
     userUpdated: false,
 
+    isCollapsed: false,
```

Comment:
what if we let it be null until its loaded? that way we should be able to make the toggle appears only when its ready isnt it?

Reply:
in order for to set the value to null until loaded, i need to retrieve the information from the code panel component on whether the code view is loaded. Anyways, as mentioned in the above discussion, this can be done as a separate feature.

Reply:
@eugenepeh ```this.isCollapsed = false``` is necessary to ensure that the button is reset to ```Collapse all``` when the code panel is closed and re-opened for another author. Otherwise, if the button was ```Expand all``` when the code panel is closed, when it is opened again for another author, button remains as ```Expand all``` and is not reset to ```Collapse all```.

Reply:
I see. Good point.